aws-organizations-policy/src/main/java/software/amazon/organizations/policy/ReadHandler.java

```diff
@@ -39,9 +41,17 @@ public ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                 .handleError((organizationsRequest, e, proxyClient1, model1, context) ->
                                  handleErrorInGeneral(organizationsRequest, e, proxyClient1, model1, context, logger, PolicyConstants.Action.DESCRIBE_POLICY, PolicyConstants.Handler.READ))
                 .done(describePolicyResponse -> {
+                    try {
+                        model.setContent(Translator.convertStringToObject(describePolicyResponse.policy().content()));
```

Comment:
Readhandler always returns Json object irrespective of the input format, is that the expected behavior?

Reply:
synced offline